lib/publiccloud/provider.pm

```diff
@@ -162,7 +162,7 @@ sub create_ssh_key {
     record_info($alg, "The $alg key will be generated.");
     if (script_run('test -f ' . $self->ssh_key) != 0) {
         assert_script_run('SSH_DIR=`dirname ' . $self->ssh_key . '`; mkdir -p $SSH_DIR');
-        assert_script_run('ssh-keygen -t ' . $alg . ' -q -N "" -C "" -m pem -f ' . $self->ssh_key);
+        assert_script_run('ssh-keygen -b 2048 -t ' . $alg . ' -q -N "" -C "" -m pem -f ' . $self->ssh_key);
```

Comment:
Please why this 2048 value? 
From `man ssh-keygen`, for Ed25519 keys `-b` is ignored, and for RSA default is 3072 bits.

Reply:
True, I will try to get rid of this one.